src/internal/clusterstate/v2.6.0/pfsdb.go

```diff
@@ -19,6 +19,72 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/pfs"
 )
 
+var schemaUpdate = `
+       ALTER TABLE pfs.commits ADD CONSTRAINT fk_col_commit
+                  FOREIGN KEY(commit_id)
+                  REFERENCES collections.commits(key)
+                  ON DELETE CASCADE
+`
+
+func SetupCommitProvenanceV01(ctx context.Context, tx *pachsql.Tx) error {
+	_, err := tx.ExecContext(ctx, schemaUpdate)
+	return errors.EnsureStack(err)
+}
+
+func SetupCommitProvenanceV0(ctx context.Context, tx *pachsql.Tx) error {
+	_, err := tx.ExecContext(ctx, schema)
+	return errors.EnsureStack(err)
+}
+
+// TODO(acohen4): verify how postgres behaves when does altering a table affect foreign key constraints?
+var schema = `
```

Comment:
Where did schema string live previously?  Is this new, or copy pasted?

Reply:
https://github.com/pachyderm/pachyderm/pull/9199/files#diff-4b93de8aecf18588c7903d6c07e15942c5b29a5c4ce9e4aa1a1ec058b15934d6L152